### ISSUE TITLE
"pull to refresh" behavior with mouse wheel scrolling

### ISSUE THREAD

**Issue by kjk** [wailsapp/wails#869]
This is windows 11, beta 12.
Default template with simple addition to add scrollable content, test repro at https://github.com/kjk/wailstest

If you mouse wheel scroll (in my case it's scrolling with touchpad on a laptop) the window has "pull to refresh" behavior, which is probably not the right default for a desktop app.

Not sure if it can be disabled on the engine level.
In CSS it can be fixed with:
```css
body {
      overscroll-behavior: none;
}
```

https://w3bits.com/prevent-chrome-pull-to-refresh-css/
https://developers.google.com/web/updates/2017/11/overscroll-behavior

**Comment by kjk** [comment]
Testing further: it doesn't happen when I use scroll-wheel of a mouse.

It does happen when I use 2 finger scrolling up/down on a touchpad on my thinkpad.

My guess the difference is that mouse sends scroll events and touchpad sends `touchmove` events which makes the browser apply phone-like behavior, which is to use "pull to refresh" unless explicitly disabled with `overscroll-behavior`.

**Comment by leaanthony** [comment]
Do you know of any gesture/touch emulators for windows?

**Comment by leaanthony** [comment]
It's set to false, so I'm not sure what else to do on this... https://github.com/wailsapp/wails/commit/b61fd16936726cbf42b0bf5ad1823632bcc3c788

**Comment by kjk** [comment]
I would recommend updating template with:
```html
body {
      overscroll-behavior: none;
}
```

It took me a while to find this css property even though I knew I solved similar problem in the past.

I just checked and Edge Browser has the same behavior on Win 11 even though it's based on Chrome and chrome doesn't do that.

I don't think it should but maybe it's confusing a laptop with touch screen (thinkpad with touch screen) for a phone or tablet.

Alternatively, it's worth documenting for others.
